tests/test_core.py

```diff
@@ -7592,6 +7602,33 @@ def test_minimal_runtime_global_initializer(self):
     self.maybe_closure()
     self.do_run(open(path_from_root('tests', 'test_global_initializer.cpp')).read(), 't1 > t0: 1')
 
+  @no_fastcomp()
```

Comment:
Does this function work with fastcomp?

Reply:
Derek told me to not bother with making the feature work for fastcomp at all.